app/app.go

```diff
@@ -638,6 +647,30 @@ func New(
 	// this line is used by starport scaffolding # stargate/app/beforeInitReturn
 	app.scopedWasmKeeper = scopedWasmKeeper
 
+	// create the simulation manager and define the order of the modules for deterministic simulations
```

Comment:
+1 safety first

Reply:
what do you mean by +1 safety first?

Reply:
the fact of that comment, regarding **deterministic simulations** - tests and simulations in that field are extremely important to sustain a global common consensus. 

i've seen this comment being added, and gave @faddat +1 reputation points for him ^^

Reply:
yes, I think that this conversation is resolved